hdfs3/core.py

```diff
@@ -413,6 +421,12 @@ def tail(self, path, size=None):
             f.seek(length - size)
             return f.read(size)
 
+    def head(self, path, size=None):
```

Comment:
This was done for the CLI, which doesn't deal nicely with missing arguments going to methods where we don't know the necessary parameter types.